test/functionalTest/cases/1048_subscription_cache/three_subscriptions.test

```diff
@@ -299,10 +299,10 @@ Date: REGEX(.*)
     "requestSemaphoreWaitingTime": "REGEX(\d+.\d+)",
     "statisticsRequests": "2",
     "subCache": "REGEX([0-9a-f, ]{50})",
-    "subCacheInserts": "2",
+    "subCacheInserts": "3",
```

Comment:
I think the +1 in this test is because the SHELL included a isPattern=false subscription that, with the new changes, now it taken into account in the cache. Is that correct?